code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -3302,8 +3302,178 @@
 	description = "An alcoholic beverage made from lightly fermented Sareszhi berries, considered an upper class delicacy on Moghes. Significant butanol content indicates intoxicating effects on Unathi."
 	color = "#bf8fbc"
 	strength = 20
-	taste_description = "grape soda"
+	taste_description = "berry juice"
 
 	glass_icon_state = "sarezhiglass"
 	glass_name = "glass of Sarezhi Wine"
-	glass_desc = "It tastes like flat grape soda. Is this supposed to be alcoholic?"
+	glass_desc = "It tastes like plain berry juice. Is this supposed to be alcoholic?"
+
+//Kaed's Unathi Cocktails
+//=======
+//What an exciting time we live in, that lizards may drink fruity girl drinks.
+/datum/reagent/butanol/moghesmargarita
+	name = "Moghes Margarita"
+	id = "moghesmargarita"
+	description = "A classic human cocktail, now ruined with cactus juice instead of tequila."
+	color = "#8CFF8C"
+	strength = 30
+	taste_description = "lime juice"
+
+	glass_icon_state = "cactusmargarita"
+	glass_name = "glass of Moghes Margarita"
+	glass_desc = "A classic human cocktail, now ruined with cactus juice instead of tequila."
+	glass_center_of_mass = list("x"=16, "y"=8)
+
+/datum/reagent/butanol/cactuscreme
+	name = "Cactus Creme"
+	id = "cactuscreme"
+	description = "A tasty mix of berries and cream with xuizi juice, for the discerning unathi."
+	color = "#664300"
+	strength = 15
+	taste_description = "creamy berries"
+
+	glass_icon_state = "cactuscreme"
+	glass_name = "glass of Cactus Creme"
+	glass_desc = "A tasty mix of berries and cream with xuizi juice, for the discerning unathi."
+	glass_center_of_mass = list("x"=17, "y"=8)
+
+/datum/reagent/butanol/bahamalizard
+	name = "Bahama Lizard"
+	id = "bahamalizard"
+	description = "A tropical cocktail containing cactus juice from Moghes, but no actual alcohol."
+	color = "#FF7F3B"
+	strength = 15
+	taste_description = "lime and orange"
+
+	glass_icon_state = "bahamalizard"
+	glass_name = "glass of Bahama Lizard"
+	glass_desc = "A tropical cocktail containing cactus juice from Moghes, but no actual alcohol."
+	glass_center_of_mass = list("x"=16, "y"=5)
+
+/datum/reagent/butanol/lizardphlegm
+	name = "Lizard Phlegm"
+	id = "lizardphlegm"
+	description = "Looks gross, but smells fruity."
+	color = "#8CFF8C"
+	strength = 20
+	taste_description = "fruity 'n creamy"
```

Comment:
This might be a little awkward.

"You taste fruity 'n creamy."

Reply:
That's literally the taste of boogers, with sweet changed to fruity.  I can change it, but I should point out that boogers are described as 'sweet 'n creamy'